lib/container_orchestrator/object_definition.rb

```diff
@@ -174,14 +169,10 @@ def messaging_environment
         {:name => "MESSAGING_PORT", :value => ENV["MESSAGING_PORT"]},
         {:name => "MESSAGING_TYPE", :value => ENV["MESSAGING_TYPE"]},
         {:name => "MESSAGING_SSL_CA", :value => ENV["MESSAGING_SSL_CA"]},
-        {:name => "MESSAGING_KEYSTORE", :value => ENV["MESSAGING_KEYSTORE"]},
-        {:name => "MESSAGING_KEYSTORE_PASSWORD", :value => ENV["MESSAGING_KEYSTORE_PASSWORD"]},
-        {:name      => "MESSAGING_HOSTNAME",
-         :valueFrom => {:secretKeyRef=>{:name => "kafka-secrets", :key => "hostname"}}},
-        {:name      => "MESSAGING_PASSWORD",
-         :valueFrom => {:secretKeyRef=>{:name => "kafka-secrets", :key => "password"}}},
-        {:name      => "MESSAGING_USERNAME",
-         :valueFrom => {:secretKeyRef=>{:name => "kafka-secrets", :key => "username"}}}
```

Comment:
Not sure how this was overlooked for so long but if a different secret name other than the default `kafka-secrets` is specified in the CR, this will fail https://github.com/ManageIQ/manageiq-pods/blob/master/manageiq-operator/api/v1alpha1/manageiq_types.go#L167-L169